README.md

```diff
@@ -305,6 +305,15 @@ packaging as Debian:
 $ sudo apt-get install ripgrep
 ```
 
+If you're an **Ubuntu Xenial (16.04)** user, ripgrep source is
+[available](https://launchpad.net/ubuntu/+source/rust-ripgrep) and the packaging 
+as Debian package is available in [PPAs](https://launchpad.net/~jerem-ferry/+archive/ubuntu/rust) .
+So , add the custom PPA and install it as :
+
+```
+$ sudo apt-get install ripgrep
+```
+
```

Comment:
Please fix the formatting here. You have whitespace around punctuation and lines exceeding 79 columns.